### PR TITLE
docs: recommendation APIM enable caching to reduce latency

### PR DESCRIPTION
**Pull request by lvlindv**
# Overview/Summary

APRL recommendation to improve latency with caching for APIM

## Related Issues/Work Items

https://dev.azure.com/CSUSolEng/Accelerator%20-%20AI/_sprints/taskboard/Accelerator%20-%20AI%20Team/Accelerator%20-%20AI/December%20'24?workitem=38369

### Breaking Changes

1. Recommendation added APIM

## As part of this pull request I have

- [X] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/contributing) and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/pulls)
- [X] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/issues) or ADO Work Items (Internal Only)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/tree/main)
- [X] Ensured PR tests are passing
- [X] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries
- [X] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by oZakari — DISMISSED** [review]
LGTM

**Review by ejhenry — COMMENTED** [review]
Thanks @lvlindv. Please include the kql file for the new recommendation. If you don't currently have an ARG query for the recommendation you can use the place holder content listed in the contribution guide.

**Comment by lvlindv** [comment]
@ejhenry added kql file

**Review by oZakari — APPROVED** [review]

**Comment by aarthiem** [comment]
@ejhenry - The new recommendation looks good. Only comment is to have the impact as Medium instead of High.

**Review by aarthiem — APPROVED** [review]
Looks good to me and please set the impact to Medium

File: azure-resources/ApiManagement/service/recommendations.yaml [hunk 1]
```diff
@@ -71,3 +71,22 @@
   learnMoreLink:
     - name: Setting up auto-scale for Azure API Management
       url: "https://learn.microsoft.com/azure/api-management/api-management-howto-autoscale"
+
+- description: Enable caching to improve performance in Azure API Management
+  aprlGuid: badd9a1f-222a-498d-ab84-1f14892b1c1b
+  recommendationTypeId: null
+  recommendationControl: HighAvailability
+  recommendationImpact: High
```
Comment (ejhenry) [inline azure-resources/ApiManagement/service/recommendations.yaml:79]:
```suggestion
  recommendationImpact: Medium
```

**Review by oZakari — APPROVED** [review]

**Review by ejhenry — APPROVED** [review]
LGTM

**Review by oZakari — APPROVED** [review]

**Pull request merged**